test/utils.py

```diff
@@ -79,3 +81,24 @@ def wait_file(dir_name, file_pattern, exclude_list, timeout_sec=1):
         iter_count = iter_count + 1
 
     return ""
+
+
+def kill_instance(instance_file_name):
+    # Kill tarantool instance by file config name.
+    pid_instance = subprocess.run(["pgrep", instance_file_name], capture_output=True)
+    if len(pid_instance.stdout) != 0:
+        os.kill(int(pid_instance.stdout), signal.SIGKILL)
+
+
+def find_free_port():
+    # Return free port in range [49152, 65535], else return None.
+    for port in range(49152, 65536):
+        sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        try:
+            sock.bind(("0.0.0.0", port))
+            sock.close()
+            return port
+        except Exception:
+            pass
```

Comment:
BTW, I would highlight a problem I meet a long time ago on Travis CI: https://github.com/tarantool/test-run/commit/be4ca7dad54a3028d36e87cd1fca9bd0d0da4d4a. I can't precisely describe this old situation, but what I remember:

* IPv6 stack was enabled, but was not workable.
* I decided to consider a port as in use if it is in use either in IPv4 or in IPv6 as extra protection from port clashing errors. I meet the similar problem later in https://github.com/tarantool/test-run/pull/197.

As a general recommendation: always keep in the mind, whether you're working with IPv4, IPv6 or both in a particular place of the code. Prefer to pass addresses, not just ports to avoid ambiguity. Aware that `getaddrinfo()` may return several addresses and if both IPv4 and IPv6 ports have a listening server, the choice is arbitrary (but usually IPv6). So set host/ip/protocol explicitly, where applicable.

Reply:
It seems, it is irrelevant now.